Mathlib/RingTheory/Jacobson.lean

```diff
@@ -514,6 +514,7 @@ theorem isMaximal_comap_C_of_isMaximal [Nontrivial R] (hP' : ∀ x : R, C x ∈
 set_option linter.uppercaseLean3 false in
 #align ideal.polynomial.is_maximal_comap_C_of_is_maximal Ideal.Polynomial.isMaximal_comap_C_of_isMaximal
 
+attribute [instance 1000] Algebra.id in
```

Comment:
`  refine' RingHom.IsIntegral.tower_bot φ (algebraMap (R[X] ⧸ P) (Localization M')) _ _` fixes this one.